### DISCUSSION TITLE
Welcome to mobile Discussions!

### DISCUSSION THREAD

**Discussion by josh-griffin** [msupply-foundation/mobile#3713] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by josh-griffin** [comment]
what about another one

**Reply by josh-griffin** [reply]
thats pretty cooool

**Reply by josh-griffin** [reply]
i tried to upvote my own thing and it gave me -1 is that because its my own or because all my upvotes are actually downvotes?

**Reply by DhanyaHerath** [reply]
Oh same. Is that a bug? It goes from 0 to -1 both ways...but the +1 side isn't red so that's nice

**Reply by josh-griffin** [reply]
I think they only allow cool people to upvote 😢

**Reply by DhanyaHerath** [reply]
😢
